rust/osm2lanes/src/lib.rs

```diff
@@ -216,6 +225,36 @@ mod tests {
         output: Vec<LaneSpec>,
         #[serde(rename = "skip_rust")]
         skip: Option<bool>,
+        comment: Option<String>,
+    }
+
+    impl TestCase {
+        fn print(&self) {
+            if !self.way_id.is_none() {
+                println!(
+                    "For input (example from https://www.openstreetmap.org/way/{}) with {}:",
+                    self.way_id.unwrap(),
+                    self.driving_side.to_tla(),
+                );
+            } else if !self.link.is_none() {
+                println!("For input (example from {}):", self.link.as_ref().unwrap());
+            }
+            if let Some(comment) = self.comment.as_ref() {
+                println!("        Comment: {}", comment);
+            }
+            for (k, v) in self.tags.map() {
+                println!("    {} = {}", k, v);
+            }
+        }
+    }
+
+    impl DrivingSide {
+        const fn to_tla(&self) -> &'static str {
```

Comment:
I'm a bit lost on the acronyms.. LHD is left handed driving? TLA is?

Reply:
TLA = three-letter acronym
it's funnier to leave it abbreviated, but I will expand it out :P

Reply:
Oh wow there's some irony here...

Reply:
Sorry for the off topic, but this is funniest smart joke I ever seen on the GitHub. Hope you don't mind if I screen it.